### PR TITLE
css: add margin-top to ResultSet

### PR DESCRIPTION
**Pull request by mistercrunch**
### CATEGORY

Choose one

- [ ] Bug Fix
- [x] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
See images

#### Before
Screen Shot 2019-06-08 at 8 40 17 PM
Screen Shot 2019-06-08 at 8 40 10 PM

#### After
Screen Shot 2019-06-08 at 8 38 24 PM
Screen Shot 2019-06-08 at 8 38 20 PM

### CODE REVIEW

File: superset/assets/src/SqlLab/components/ResultSet.jsx [hunk 1]
```diff
@@ -73,6 +73,7 @@ export default class ResultSet extends React.PureComponent {
     // when new results comes in, save them locally and clear in store
     if (this.props.cache && (!nextProps.query.cached)
       && nextProps.query.results
+      && nextProps.query.results.data
```
Comment (mistercrunch) [inline superset/assets/src/SqlLab/components/ResultSet.jsx]: FYI: Unrelated bug I hit while doing CSS tweaks

**Review by etr2460 — APPROVED** [review]
lgtm

I'm curious about the structure here and if there's a reason why we wouldn't just want to add padding to the `tab-pane` div by default so that each possible child element wouldn't need to have a margin. Does it not play nicely with the table elements?

**Comment by mistercrunch** [comment]
I thought about the tradeoffs of padding the tab-pane a little, and though maybe we wanted more density like for data tables/previews. Then I thought that we don't want left or right margins for the layout to align.

I can to the conclusion that it's for the children to decide how they want to use the space.

I'm no designer and don't know much about structuring CSS properly. Happy to take a different approach.

**Comment by etr2460** [comment]
That makes sense to me, thanks for the context!

**Pull request merged**